### ISSUE TITLE
Hash FMX demo crashes on Android 32 bit in release mode

### ISSUE THREAD

**Issue by MHumm** [MHumm/DelphiEncryptionCompendium#26]
The Hash FMX demo app crashes when compiled as release mode Android 32 app. When compiled as Android 32 bit debug mode app it works. The reason is not known yet but it will be investigated.

Some user reported that this code is sufficient to crash it aöready:
`uses
 DECHash

var
  Hash: THash_MD5;
begin

  Hash := THash_MD5.Create;
  Try
    sleep(1000);
  Finally
    Hash.Free;
  End;`

**Comment by bdwilton** [comment]
Yes just gave Cipher_FMX test under 10.4.2 and 32-bit debug mode works while release hangs on start-up

**Comment by MHumm** [comment]
I guess this is a duplicate of this one:
[https://github.com/MHumm/DelphiEncryptionCompendium/issues/28](https://github.com/MHumm/DelphiEncryptionCompendium/issues/28)
